topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md

```diff
@@ -218,11 +211,9 @@ As we directly see, the peak file lacks `chr` before any chromosome number. But
 
 > ### {% icon hands_on %} Hands-on: View end of file
 >
-> 1. Search for the **Select last** {% icon tool %} tool in the tool panel on the left
-> 2. Select the following settings
->     - **Text file** to our peak file `GSE37268_mof3.out.hpeak.txt`
->     - **Operation**: `Keep last lines`
->     - **Number of lines**: Choose a value, e.g. `100`
+> 1. **Select last** {% icon tool %}: Run **Select last lines from a dataset** with the following settings:
+>     - **Select last**: Choose a value, e.g. `100`
+>     - **from**: Our peak file `GSE37268_mof3.out.hpeak.txt`
```

Comment:
That's not correct, this tutorial is using the tool [tp_tail_tool](https://github.com/bgruening/galaxytools/blob/master/tools/text_processing/text_processing/tail.xml) ("Select last lines from a dataset (tail)") from the [text_processing](https://toolshed.g2.bx.psu.edu/view/bgruening/text_processing/) ToolShed repository, not the tool [Show tail1](https://github.com/galaxyproject/galaxy/blob/dev/tools/filters/tailWrapper.xml) ("Select last lines from a dataset") coming together with Galaxy.
See https://github.com/galaxyproject/training-material/blob/master/topics/introduction/tutorials/galaxy-intro-peaks2genes/tools.yaml#L11